specification/cognitiveservices/data-plane/AnomalyDetector/stable/2022-08-23/MultivariateAnomalyDetector.json

```diff
@@ -1,1222 +1,987 @@
-          "example": 0.8,
-          "minimum": 0,
-          "maximum": 1
-        },
-        "score": {
-          "type": "number",
-          "description": "Raw score from the model.",
-          "example": 0.3,
-          "minimum": 0,
-          "maximum": 2
-        },
-        "interpretation": {
-          "type": "array",
-          "items": {
-            "$ref": "#/definitions/AnomalyInterpretation"
-          }
-        }
+      "DetectionResult": {
+          "required": ["resultId", "results", "summary"],
+          "properties": {
+              "resultId": {
+                  "type": "string",
+                  "format": "uuid",
+                  "example": "45aad126-aafd-11ea-b8fb-d89ef3400c5f"
+              },
+              "summary": {
+                  "description": "Multivariate anomaly detection status.",
+                  "$ref": "#/definitions/DetectionResultSummary"
+              },
+              "results": {
+                  "type": "array",
+                  "description": "Detection result for each timestamp.",
+                  "items": {
+                      "$ref": "#/definitions/AnomalyState"
+                  }
+              }
+          },
+          "type": "object",
+          "description": "Detection results for the given resultId."
       },
-      "type": "object"
-    },
-    "LastDetectionRequest": {
-      "required": [
-        "detectingPoints",
-        "variables"
-      ],
-      "properties": {
-        "variables": {
-          "type": "array",
-          "description": "variables",
-          "items": {
-            "$ref": "#/definitions/VariableValues"
-          }
-        },
-        "detectingPoints": {
-          "type": "integer",
-          "description": "number of timestamps on which the model detects",
-          "example": 2
-        }
+      "DetectionResultSummary": {
+          "required": ["setupInfo", "status"],
+          "properties": {
+              "status": {
+                  "description": "Status of detection results. One of CREATED, RUNNING, READY, and FAILED.",
+                  "$ref": "#/definitions/DetectionStatus"
+              },
+              "errors": {
+                  "type": "array",
+                  "description": "Error message when detection is failed.",
+                  "items": {
+                      "$ref": "#/definitions/ErrorResponse"
+                  }
+              },
+              "variableStates": {
+                  "type": "array",
+                  "example": [{
+                          "effectiveCount": 1441,
+                          "endTime": "2019-04-02T00:00:00Z",
+                          "filledNARatio": 0,
+                          "startTime": "2019-04-01T00:00:00Z",
+                          "variable": "ad_input"
+                      }, {
+                          "effectiveCount": 1441,
```

Comment:
@mikekistler do we have any preference for using Swagger's native "example" field vs. "x-ms-examples"?

Reply:
Just as Mike said, we'll keep "example" in this version.